### PR TITLE
fix: moved delete all sample data from account settings

### PR DESCRIPTION
**Pull request by Riddhiagrawal001**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
image

## Checklist

- [X] I ran `npm run re:build`
- [X] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/screens/Order/OrderUIUtils.res [hunk 1]
```diff
@@ -74,15 +75,67 @@ module GenerateSampleDataButton = {
...
+        <ACLButton
+          access={userPermissionJson.operationsManage}
+          buttonType={Secondary}
+          buttonSize={XSmall}
+          text="Generate Sample Data"
+          customButtonStyle="!rounded-l-md !rounded-none"
+          onClick={_ => generateSampleData()->ignore}
+          leftIcon={CustomIcon(<Icon name="plus" size=13 />)}
+        />
+        <div
```
Comment (JeevaRamu0104) [inline src/screens/Order/OrderUIUtils.res:133]: use ACLDiv

Comment (Riddhiagrawal001) [reply src/screens/Order/OrderUIUtils.res]: Updated

**Review by gitanjli525 — APPROVED** [review]

**Review by gitanjli525 — CHANGES_REQUESTED** [review]
We have to remove "delete sample data" from Account Settings also?

**Comment by Riddhiagrawal001** [comment]
@gitanjli525 
Already handled in https://github.com/juspay/hyperswitch-control-center/pull/1027

**Comment by gitanjli525** [comment]
Can we fix this ?
image

**Comment by gitanjli525** [comment]
Can we position center the bin icon?
image

**Review by JeevaRamu0104 — APPROVED** [review]

**Comment by Riddhiagrawal001** [comment]
@gitanjli525 
image

**Review by JeevaRamu0104 — APPROVED** [review]

**Review by gitanjli525 — APPROVED** [review]

**Review by PritishBudhiraja — APPROVED** [review]

**Pull request merged**